test/controllers/ControllerBehaviours.scala

```diff
@@ -123,7 +123,7 @@ trait ControllerBehaviours {
     redirectNextPage(call, defaultUserAnswers, form: _*)
 
   def redirectToPage(call: => Call, page: => Call, userAnswers: UserAnswers, form: (String, String)*): Unit =
-    s"redirect to page ${page.url} with form $form" in {
```

Comment:
stopped evaluating the play Call in the test name to prevent any issues in the future